references/cli/portforward.go

```diff
@@ -124,7 +124,7 @@ func (o *VelaPortForwardOptions) Init(ctx context.Context, cmd *cobra.Command, a
 	o.Cmd = cmd
 	o.Args = argsIn
 
-	env, err := GetEnv(o.Cmd)
+	env, err := GetFlagEnvOrCurrent(o.Cmd, o.VelaC)
```

Comment:
What's `VelaC`?

Reply:
It's common.Args. I add this in func because `env` won't be store locally and this func need an K8s client.